libkbfs/folder_branch_ops.go

```diff
@@ -4494,6 +4502,24 @@ func (fbo *folderBranchOps) GetUpdateHistory(ctx context.Context,
 	return history, nil
 }
 
+// GetEditHistory implements the KBFSOps interface for folderBranchOps
+func (fbo *folderBranchOps) GetEditHistory(ctx context.Context,
+	folderBranch FolderBranch) (edits TlfWriterEdits, err error) {
+	fbo.log.CDebugf(ctx, "GetWriterEdits")
+	defer func() { fbo.deferLog.CDebugf(ctx, "Done: %v", err) }()
+
+	if folderBranch != fbo.folderBranch {
+		return nil, WrongOpsError{fbo.folderBranch, folderBranch}
+	}
+
+	head, err := fbo.getMDForExternalUse(ctx)
```

Comment:
You can just call `getMDForReadHelper` directly here and revert the name change, right?